File: Java/src/e_03.java
public class e_03 {
    public static void main(String[] args) {
        // Declarar las variables y asignarles valores
        int x = 3;
        double y = 8.32;

        // Mostrar el tipo de las variables antes de la conversión
        System.out.println("Tipo de x antes de la conversión: " + obtenerTipo(x));
        System.out.println("Tipo de y antes de la conversión: " + obtenerTipo(y));

        // Convertir el tipo de las variables a cadenas de caracteres
        String strX = String.valueOf(x);
        String strY = String.valueOf(y);

        // Mostrar el tipo de las variables después de la conversión
        System.out.println("Tipo de x después de la conversión: " + obtenerTipo(strX));
        System.out.println("Tipo de y después de la conversión: " + obtenerTipo(strY));
    }

    // Función para obtener el tipo de una variable como cadena
    private static String obtenerTipo(Object variable) {
        return variable.getClass().getSimpleName();
    }

}

File: JS/01-10/e_03.js
// Escribe una serie de instrucciones en JS que permita declarar dos variables "x" e "y" asignando los valores 3 y 8.32, 
// luego convertir el tipo de estas variables a una cadena de caracteres. 
// Al final, el programa debe mostrar el tipo de estas variables antes y después de la conversión


// Declarar las variables y asignarles valores
let x = 3;
let y = 8.32;

// Mostrar el tipo de las variables antes de la conversión
console.log("Tipo de x antes de la conversión:", typeof x);
console.log("Tipo de y antes de la conversión:", typeof y);

// Convertir el tipo de las variables a cadenas de caracteres
x = String(x);
y = String(y);

// Mostrar el tipo de las variables después de la conversión
console.log("Tipo de x después de la conversión:", typeof x);
console.log("Tipo de y después de la conversión:", typeof y);
